src/Tasks/JTask.php

```diff
@@ -69,6 +69,11 @@ public function __construct($params = array())
 		$this->loadConfiguration($params);
 		$this->determineOperatingSystem();
 		$this->determineSourceFolder();
+
+		// Registers the application to run Robo commands
```

Comment:
This needs to be above the configuration load.  Check how it's done in https://github.com/joomla-projects/jorobo/blob/container-test/src/Tasks/JTask.php#L69-L77